vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -369,6 +363,18 @@ void TouchEventHandler::UpdatePointersInViews(
 }
 
 winrt::Microsoft::ReactNative::JSValue TouchEventHandler::GetPointerJson(const ReactPointer &pointer, int64_t target) {
+  // defines button payoad, follows https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button
```

Comment:
nit: typo, "payload"
nit: remove the en-US segment in urls, it will take the user to the right page for the user's language/region.